### ISSUE TITLE
Fix S5693 FN: Support generic attributes

### ISSUE THREAD

**Issue by cristian-ambrosini-sonarsource** [SonarSource/sonar-dotnet#6384]
### Description

FN for S5693 (RequestsWithExcessiveLength).

Using C#11 generic attributes, the rule is not raising an issue.

### Repro steps

```csharp
           internal class MyClass
          {
              [GenericAttribute<int>(8_000_001)] // FN
              public void MyMethod() { }
          }
```

### Expected behavior

The rule should raise an issue.

### Actual behavior

The rule does not raise an issue.

**Comment by csaba-sagi-sonarsource** [comment]
The rule actually does not support derived attributes at all, and it will not be fixed as it is an edge case and would have a heavy performance impact on the rule. The decision was documented in https://github.com/SonarSource/sonar-dotnet/pull/6452
